include/SSVOpenHexagon/Global/Config.hpp

```diff
@@ -19,6 +20,39 @@ class Trigger;
 
 namespace hg::Config
 {
+struct Version
+{
+    int major;
+    int minor;
+    int micro;
+
+    Version& operator=(const Version& o)
+    {
+        major = o.major;
+        minor = o.minor;
+        micro = o.micro;
+        return *this;
+    }
+
+    auto operator<(const Version& rhs) const
+    {
+        return std::tie(major, minor, micro) <
+               std::tie(rhs.major, rhs.minor, rhs.micro);
+    }
+
+    auto operator>(const Version& rhs) const
+    {
+        return std::tie(major, minor, micro) >
+               std::tie(rhs.major, rhs.minor, rhs.micro);
+    }
+
+    bool operator==(Version other) const
+    {
+        return (major == other.major) && (minor == other.minor) &&
+               (micro == other.micro);
+    }
+};
+const Version GAME_VERSION = Version{2, 0, 3};
```

Comment:
```suggestion
constexpr Version GAME_VERSION{2, 0, 3};
```

Reply:
constexpr is new to me. I don't know what this is but I will take your word for it.

Reply:
There's now a compiler error when this is applied. Again I have no knowledge on constexpr so I don't know what is going on with this error.